recipes/coreutils/coreutils.inc

```diff
@@ -0,0 +1,38 @@
+DESCRIPTION = "The GNU Core Utilities provide the basic file, shell and text \
+manipulation utilities. These are the core utilities which are expected to exist on \
+every system."
+LICENSE = "GPLv3+"
+
+COMPATIBLE_HOST_ARCHS = ".*linux"
+
+inherit autotools gettext pkgconfig
+
+require conf/fetch/gnu.conf
+
+SRC_URI = "${GNU_MIRROR}/${PN}/${PN}-${PV}.tar.xz"
+
+EXTRA_OECONF:>native += " --without-gmp"
+EXTRA_OECONF:>machine += " --enable-install-program=arch --enable-single-binary"
+EXTRA_OECONF:>sdk += " --enable-install-program=arch"
+
+DEPENDS += "gettext libpthread libgmp libcap"
+
+RECIPE_FLAGS += "ssl"
+EXTRA_OECONF += "${EXTRA_OECONF_SSL}"
+EXTRA_OECONF_SSL = "--without-openssl"
+EXTRA_OECONF_SSL:USE_ssl = "--with-openssl=yes"
+DEPENDS:>USE_ssl = " libssl"
+
+inherit auto-package-utils
+
+AUTO_PACKAGE_UTILS = "[ chown dd expr install mkfifo od readlink sha384sum stty tr uptime \
```

Comment:
Maybe use `bracket:[` and see if that will work.  That should give you coreutils-bracket instead of coreutils-[ which simply looks too dangerous.